pyproject.toml

```diff
@@ -31,8 +31,7 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.10,<4.0"
 ansys-openapi-common = "^2.0.0"
-# Ensure PIP_INDEX_URL is not set in CI when building on release branches.
-ansys-grantami-serverapi-openapi = "~=5.0"
+ansys-grantami-serverapi-openapi = "5.1.0rc0"
```

Comment:
Is this exact pin just for the RC? I suspect we need to specify the rc0 to get it to match for now, but once we release we probably want `~= 5.1.0`